public/css/main.css

```diff
@@ -130,6 +130,12 @@ button {
 	box-shadow:0 3px 12px -1px rgba(0,0,0,.2);
 }
 
+button:disabled,
+button[disabled] {
```

Comment:
you shouldn't need the button[disabled] line as well as the :disabled selector! Unless my CSS is bad and you needed to override something haha